WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/BloggingPromptsParentViewModel.kt

```diff
@@ -1,10 +1,31 @@
 package org.wordpress.android.ui.bloggingprompts
 
+import androidx.annotation.VisibleForTesting
 import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import org.wordpress.android.WordPress
+import org.wordpress.android.fluxc.model.SiteModel
+import javax.inject.Inject
 
-class BloggingPromptsParentViewModel : ViewModel() {
-    fun onOpen() = Unit
+@HiltViewModel
+class BloggingPromptsParentViewModel @Inject constructor(
```

Comment:
**Current Reasoning.** 
The `BloggingPromptsParentViewModel` is used in the both `BloggingPromptsParentFragment` and `BloggingPromptsActivity`. I matched the name of the viewmodel with the name of the fragment instead of the activity. I'm also planning to introduce `BloggingPromptsListViewModel` later to handle the fetching and rendering of prompts. 

Group of Classes 1:
- BloggingPromptsParentViewModel
- BloggingPromptsParentFragment
- BloggingPromptsActivity

Group of Classes 2:
- BloggingPromptsListViewModel (To be Created) 
- BloggingPromptsListFragment

**New Approach.** 

Another approach is to share the same viewmodel across all three classes (Activity, ParentFragment, ListFragment), at which point, the name you suggested `BloggingPromptsListViewModel` might match more. This might also make it easier to handle error states as noticed in the analytics. 

Group of Classes:
- BloggingPromptsListViewModel
- BloggingPromptsParentFragment
- BloggingPromptsActivity
- BloggingPromptsListFragment

I'll go ahead with this approach then. Let me know if you feel otherwise.  Naming is tricky!

Reply:
I've renamed the classes. Please take a look.

Reply:
> Naming is tricky!

It's always the hardest part 🙂 IMHO the common prefix looks good, thanks